amanuensis/config-default.yaml

```diff
@@ -130,6 +130,10 @@ APPLICATION_ROOT: '/amanuensis'
 # //////////////////////////////////////////////////////////////////////////////////////
 ASSOCIATED_USER_ROLE_DEFAULT: 'METADATA_ACCESS'
 
+# //////////////////////////////////////////////////////////////////////////////////////
+# ASSOCIATED USER DEFAULT ROLE
+# //////////////////////////////////////////////////////////////////////////////////////
+NOTIFY_STATE: "DATA_DOWNLOADED"
```

Comment:
I would make this an array just in case we decide to send emails for more states in the future.